components/DeleteButton.tsx

```diff
@@ -1,51 +1,26 @@
 'use client';
-import deleteItems from '@/supabase/models/deleteItems';
-import insertSystemMessage from '@/supabase/models/messaging/insertSystemMessage';
-import selectConversationsByItemId from '@/supabase/models/messaging/selectConversationsByItemId';
 
 type DeleteButtonProps = {
-  itemId?: number;
+  targetId?: number | string;
   title: string;
```

Comment:
I know this is not you, but this should be children , type reactNode so that it could take values not a string.  